src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
How do I do that for user-defined literals (`_ml` in this case)? I tried several ways of prepending, but neither worked. Instead, I can narrow the scope of `using`. Would that suffice?